packages/driver/cypress/integration/cypress/command_queue_spec.ts

```diff
@@ -193,3 +193,34 @@ describe('src/cypress/command_queue', () => {
     })
   })
 })
+
+describe('stability', () => {
```

Comment:
Should these news tests live in the `e2e` folder instead of adding them to the integration tests? They feel more e2e-like to me

Reply:
I'll be honest, I can't make much sense of how tests are organized in the driver. I added the tests here because this was the existing command queue file. Regardless of if they are here or in the e2e folder, they both have integration as a parent directory.

I can go ahead and move them to the e2e directory, now to think of a good name......